src/collective/solr/profiles/default/controlpanel.xml

```diff
@@ -7,7 +7,7 @@
     appId="SolrSettings"
     category="Products"
     condition_expr=""
-    icon_expr="string:${portal_url}/++resource++collective.solr.images/icon.png"
+    icon_expr="string:search"
```

Comment:
@gforcada thanks for the fix. Just wondering, did you test this in Volto as well?

Reply:
nope, we might build one or two volto sites in the future, but so far we are still using  classic UI

Reply:
It doesn't seem to affect volto (which is ok):
<img width="163" alt="Screenshot 2024-05-16 at 4 31 18 PM" src="https://github.com/collective/collective.solr/assets/49014/f97db3b9-ff41-4371-9e11-d7a0769bd250">